.github/workflows/deploy-staging-ec2.yml

```diff
@@ -0,0 +1,34 @@
+on:
+  push:
+    branches:
+      - development
+name: Deploy Staging
```

Comment:
I don't think we would ever want to deploy to staging environment. Staging is for live work / live debugging. We use it to run/test the autobuilder while we are still implementing a fix/feature bc it can be faster than running the autobuilder on our local machines. Having a merge to deploy on staging doesn't make sense for our workflow